Python Part/lib/cnn.py

```diff
@@ -4,7 +4,9 @@
 class cnn_layers():
     def __init__(self, n_kernels, kernel_size):
         # Setting the shapes and sizes for convolution operation
+        threshold = 0.0002
```

Comment:
I believe this is for the compression, which is the core idea of our project, but it's not clear that it's doing compression.

Also, we need to compare the accuracy, with the compression and without it, so we see how much it got affected by it.

So, it'd be a good idea if you create separate functions that we can use to see the difference.